team-telnyx/telnyx-client-examples
# Dial using call control

Dials a number using call control by way of a new `clientCallState` field, which can be set to auto-answer.

Known issues:
- Incoming call UI flashes before auto-answer
- Sound only works on Chrome (tested in FF)
- TODO related to https://github.com/team-telnyx/webrtc/pull/46

## ✋ Manual testing

1. Run the call-center app and log in
2. Dial a phone number or SIP username you have access to.
- Verify you see the incoming call UI state (TODO: hide this state for outgoing calls.)
- Verify that incoming call is automatically answered
- Verify external number is dialed once agent incoming call is answered
3. Answer call on external number. Verify you can hear the conference call

## 🦊 Browser testing

### Desktop

### Mobile

## 📸 Screenshots
